snail-kotlin/src/main/java/com/compass/snail/Observable.kt

```diff
@@ -6,6 +6,7 @@ import android.util.Log
 import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
+import java.lang.IllegalArgumentException
```

Comment:
Can we throw a kotlin error instead

Reply:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-illegal-argument-exception/index.html
just need to manually change the import, but this is a java one under the hood